app/controllers/players_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PlayersController < ApplicationController
+	def index
+		@players = Player.all
+	end
+
+	def new 
+		@player = Player.new
+	end
+
+	def create
+		@player = Player.create(player_params)
+		redirect_to root_path
+	end
+
+	def edit
+		@player = Player.find(params[:id])
+	end
+
+	def update
+		@player = Player.find(params[:id])
+		return redirect_to root_path if @player.update(player_params)
+		redirect_to edit_player_path(params[:id])
```

Comment:
@rikukawano I've refactored this code a little bit.
* Instead of an if else statement I've changed it into something called a "guard clause". 
* Basically, we return from the action if the player updates (if @player.update), meaning that the code below it (in this case redirect_to edit_player_path) won't run. 
* If It doesn't update, then it runs the code underneath (redirect_to edit_player_path) 
* This is a common pattern that is used in Ruby on Rails, so it is good to take note
* Another example:
```
if @player.year == 2
  do_something
else 
  do_something_else
end
```
can be refactored to

```
return do_something if @player.year == 2
do_something_else
```